CONTRIBUTING.md

```diff
@@ -244,10 +225,8 @@ particular will be located at `docs/_build/html/release_notes.html`
 Please see the [Installing Qiskit Nature from
 Source](https://github.com/qiskit-community/qiskit-nature#installation)
 
-Note: Nature depends on Terra, and has optional dependence on Aer and IBM Q Provider, so
-these should be installed too. The main branch of Nature is kept working with those other element
-main branches so these should be installed from source too following the instructions at 
-the same location
+Note: Nature depends on Qiskit, and has optional dependence on Aer so
```

Comment:
```suggestion
Note: Nature depends on Qiskit, and has an optional dependence on Aer so
```
Guess it was getting later in the day when I did this this is what I had elsewhere in Opt for instance, I guess I missed adding the an here

> Note: Optimization depends on Qiskit, and has an optional dependence on Aer, so should be installed too.

Reply:
Would it not need to be `so that should/could be installed too`?

Reply:
Okay sorry, I stressed the wrong part here. The missing `"that"` is what is throwing me off.

Reply:
Ah ok `that` or `it` etc - I guess the lack of it did not - I read that as implied but it did have these there before. Ok I updated the suggestion above. Hopefully that works.